packages/react-core/src/demos/DescriptionList/DescriptionList.md

```diff
@@ -0,0 +1,25 @@
+---
+id: Description list
+section: components
+---
+
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import ThIcon from '@patternfly/react-icons/dist/esm/icons/th-icon';
+import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-circle-icon';
+import imgAvatar from '@patternfly/react-core/src/components/assets/avatarImg.svg';
+import pfIcon from '../assets/pf-logo-small.svg';
+import pfLogo from '../assets/pf-logo.svg';
```

Comment:
Doesn't seem like omitting these imports are necessary as the demo loads fine without them